maestrowf/datastructures/core/executiongraph.py

```diff
@@ -602,16 +602,21 @@ def _execute_record(self, record, adapter, restart=False):
     def write_status(self, path):
         """Write the status of the DAG to a CSV file."""
         header = "Step Name,Workspace,State,Run Time,Elapsed Time,Start Time" \
-                 ",Submit Time,End Time,Number Restarts"
+                 ",Submit Time,End Time,Number Restarts, Job ID"
         status = [header]
         keys = set(self.values.keys()) - set(["_source"])
         for key in keys:
             value = self.values[key]
+
+            jobid_str = "Job ID not available"
+            if value.jobid:
+                jobid_str = " ".join(value.jobid)
```

Comment:
As an FYI, when joining a list -- the items must be strings. It's an easy gotcha in Python.

Your current code fails here:
```
Traceback (most recent call last):
  File "/mnt/f/Code/Python/maestrowf/maestrowf/conductor.py", line 382, in main
    completion_status = conductor.monitor_study()
  File "/mnt/f/Code/Python/maestrowf/maestrowf/conductor.py", line 355, in monitor_study
    dag.write_status(os.path.split(pkl_path)[0])
  File "/mnt/f/Code/Python/maestrowf/maestrowf/datastructures/core/executiongraph.py", line 613, in write_status
    jobid_str = " ".join(value.jobid)
TypeError: sequence item 0: expected str instance, int found
[2020-09-30 23:58:04,209: INFO] Study exiting, cleaning up...
[2020-09-30 23:58:04,209: INFO] Squeaky clean!
```